pvmismatch/tests/test_pvmodule.py

```diff
@@ -40,6 +41,30 @@ def test_calc_pct_bridges():
     pvmod = PVmodule(cell_pos=pct492_bridges)
     return pvmod
 
+
+def check_same_pvconst_and_lengths(pvmod):
+    assert len(pvmod.pvcells) == 96
```

Comment:
why 96? Is it because the default PVmodule has the std96 patten?

Reply:
yes, because [`STD96` is the default module](https://github.com/SunPower/PVMismatch/blob/master/pvmismatch/pvmismatch_lib/pvmodule.py#L167), that's created when you don't specify a cell pattern, but it's confusing to hardcode it, probably better to grab the value from the default directly?